### PR TITLE
Added Automatic Instagram Profile Pic Download

### PR DESCRIPTION
**Pull request by hardikkhurana**
It can be used easily by just inputting the username of the account

# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #(issue_no)

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by hardikkhurana** [comment]
Thank you for giving me this chance I hope you like my work.

**Comment by avinashkranjan** [comment]
@hardikkhurana Kindly fix syntax error, Before making a PR check the script on your system once
ApplicationFrameHost_C1k7pWBX1e

And Few More things...
- Add a Readme file. 
- Attach a SS which shows working of your script.

File: Instagram Profile Pic Downloader/Instagram Profile Pic Downloader.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import instaloader
+username = str(input('Enter username: '))
+x = instaloader.Instaloader()
+print(x.download profile(username, profile pic_only=True))
```
Comment (avinashkranjan) [inline Instagram Profile Pic Downloader/Instagram Profile Pic Downloader.py:4]: Correct this syntax error

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Make the Changes Mentioned

**Comment by hardikkhurana** [comment]
I have solved all the errors and added SS also the readme file.

Thank You

**Comment by hardikkhurana** [comment]
@avinashkranjan Sir, have you checked my new PR?or is there any problem from my side?

**Comment by avinashkranjan** [comment]
@hardikkhurana I'm still looking into it...
ApplicationFrameHost_81sGITn3qi
Your scripts run but does nothing..

**Comment by hardikkhurana** [comment]
@avinashkranjan Sir A new folder will be created after you run the script in the same folder as the python script is stored in and in that folder there will be the profile pictures as i have also showed in the Screenshots

**Comment by avinashkranjan** [comment]
@hardikkhurana No new Folder created.. Here the proof
explorer_joJvTVxYIZ

**Comment by hardikkhurana** [comment]
@avinashkranjan I understand sir and would like to share this screen recorded video
here I am using Anaconda and Spyder

https://www.dropbox.com/s/v1rdwhua91pi3bc/Insta%20profile%20pic.mp4?dl=0

this is the link to that dropbox video 

Thank you

**Comment by avinashkranjan** [comment]
> @hardikkhurana I'm still looking into it...
@hardikkhurana I knw..I think it's a problem from my side.... That's why I said..I'm currently looking into it..! Once it gets resolved I'll merge your PR

**Review by avinashkranjan — APPROVED** [review]
LGTM...💯

**Comment by avinashkranjan** [comment]
@all-contributors please add @hardikkhurana for code and documentation

**Pull request merged**